pkgs/applications/misc/hello/default.nix

```diff
@@ -18,9 +18,16 @@ stdenv.mkDerivation rec {
       It is fully customizable.
     '';
     homepage = "https://www.gnu.org/software/hello/manual/";
-    changelog = "https://git.savannah.gnu.org/cgit/hello.git/plain/NEWS?h=v${version}";
+    changelog = "https://git.savannah.gnu.org/cgit/hello.git/plain/NEWS?h=v${this.version}";
     license = licenses.gpl3Plus;
     maintainers = [ maintainers.eelco ];
     platforms = platforms.all;
   };
-}
+
+  passthru.tests.run = runCommand "hello-test-run" {
```

Comment:
tests should be added in a separate file
maybe then this whole concept does not work anymore

Reply:
This works just fine.